### PR TITLE
Bump getdns/stubby to 1.7.0 and 0.4.0 respectively

### PR DESCRIPTION
**Pull request by rsalvaterra**
And remove obsolete/upstreamed patches.

### CODE REVIEW

**Comment by rsalvaterra** [comment]
@neheb, this appears to solve all the issues I had with truncated DNS responses.

**Comment by neheb** [comment]
@jonathanunderwood is the maintainer

File: libs/getdns/patches/010-openssl-deprecated.patch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 --- a/src/tls/val_secalgo.c
 +++ b/src/tls/val_secalgo.c
-@@ -72,6 +72,10 @@
- #include <openssl/engine.h>
+@@ -76,6 +76,10 @@
```
Comment (neheb) [inline libs/getdns/patches/010-openssl-deprecated.patch:3]: look again. This patch was upstreamed.

Comment (rsalvaterra) [reply libs/getdns/patches/010-openssl-deprecated.patch:3]: Ouch, missed it. Will fix.

**Comment by BKPepe** [comment]
Was it compile and run tested?

**Comment by rsalvaterra** [comment]
Hi,

A sexta, 11/06/2021, 21:18, Josef Schlehofer ***@***.***>
escreveu:

> Was it compile and run tested?
>
I don't make pull requests without runtime testing on at least one system.
In this case, I tested on both a TL-WDR3600 and a Redmi AC2100.

Thanks,
Rui

>

**Comment by BKPepe** [comment]
Hello,

I asked it explicitly because the PR template was removed and replaced with your own text w/o indicating that you tested it on these two routers.

Thanks for letting me know.

**Comment by neheb** [comment]
This stubby update has an annoying problem. Patch incoming.

**Comment by jonathanunderwood** [comment]
OK, yet again @neheb you have merged changes without the maintainer approval. The last time you did this I asked you not to do it again. This has now been merged with problems, and no thought given to merging onto the other active branches. 

@rsalvaterra why did you move the boilerplate for PRs from this PR, which explicitly asks you to notify the package maintainer?

Seriously, this makes me wonder if I want to continue to contribute here. It's a fairly simple process. I have previously raised with you, @neheb, not to merge PRs where the contributor has removed the boilerplate and maintainer notification. Why do you persist?

**Comment by neheb** [comment]
Hrm. I assumed you were notified. Didn't realize the PR template was removed.

**Comment by champtar** [comment]
@neheb 3 days during the week is too short for many people

**Comment by rsalvaterra** [comment]
> @rsalvaterra why did you move the boilerplate for PRs from this PR, which explicitly asks you to notify the package maintainer?

I'm sorry, I completely missed the boilerplate, won't happen again. I guess I was to eager to get this merged, as I've been waiting for more than a year for a new stubby release. It was basically unusable in OpenWrt, at least, up to now.

**Pull request merged**